std/portable/index.js

```diff
@@ -313,3 +313,57 @@ globalScope["trace"] = function(message, n) {
   if (n) message += Array.prototype.slice.call(arguments, 2, 2 + n);
   console.error("trace: " + message);
 };
+
+Object.defineProperty(Int8Array, "wrap", {
+  value: function(buffer, byteOffset, length) {
```

Comment:
could you plz add function names?
```suggestion
  value: function wrap(buffer, byteOffset, length) {
```
And for rest cases as well

Reply:
Absolutely; I can't believe I missed that.